airflow/operators/python.py

```diff
@@ -555,6 +555,11 @@ def __init__(
                 "major versions for PythonVirtualenvOperator. Please use string_args."
                 f"Sys version: {sys.version_info}. Venv version: {python_version}"
             )
+        if type(python_version) is float:
+            raise AirflowException(
```

Comment:
Better to raise a TypeError instead.

Reply:
I changed it to TypeError and also removed int type too. thank you for reveiwing

Reply:
I changed it to TypeError and also removed int type too. Thank you for reviewing